python/ray/tune/tests/test_trial_scheduler.py

```diff
@@ -976,7 +976,7 @@ def testTuneSamplePrimitives(self):
     def testTuneSampleFromError(self):
         with self.assertRaises(ValueError):
             pbt, runner = self.basicSetup(hyperparam_mutations={
-                "float_factor": tune.sample_from(lambda: 100.0)
+                "float_factor": tune.sample_from(lambda _: 100.0)
             })
```

Comment:
cc @amogkam

Reply:
Added check for parameter count in `Function` domain